contracts/ens-offchain-resolver/IExtendedResolver.sol

```diff
@@ -3,6 +3,9 @@
 pragma solidity 0.8.13;
 
 interface IExtendedResolver {
+    /**
+     * @dev implement resolve using a name and data.
+     */
```

Comment:
@petejkim - I wasnt entirely sure what the comment here should be - since this is just an interface with no implementation.

Reply:
Something like

```
    /**
     * @notice Function interface for the ENSIP-10 wildcard resolution function.
     * @param name DNS-encoded name to resolve.
     * @param data ABI-encoded data for the underlying resolution function (e.g. addr(bytes32), text(bytes32,string)).
     */
```